modules/backend/models/User.php

```diff
@@ -24,10 +24,10 @@ class User extends UserBase
      * Validation rules
      */
     public $rules = [
-        'email' => 'required|between:6,255|email|unique:backend_users',
-        'login' => 'required|between:2,255|unique:backend_users',
-        'password' => 'required:create|between:4,255|confirmed',
-        'password_confirmation' => 'required_with:password|between:4,255'
+        'email' => 'required|between:6,191|email|unique:backend_users',
+        'login' => 'required|between:2,191|unique:backend_users',
+        'password' => 'required:create|between:8,191|confirmed',
+        'password_confirmation' => 'required_with:password|between:8,191'
```

Comment:
@daftspunk @bennothommo I wonder if we need to go as deep as detecting what the default varchar length is with a special character to be parsed by the validation trait since we've introduced the config for it.

Reply:
Bit a complex idea, 191 is a reasonable length. Note password does not need this limitation because the end result is hashed

Reply:
Fair enough. @daftspunk what's the length of the hashed end result? Should we be concerned about 191 being too short? @bennothommo you can change this to be a min instead of a range.

Reply:
Password hash is 60 characters

Reply:
@LukeTowers I've changed the rules to use `min` instead. Should we do the same here: 
https://github.com/octobercms/library/blob/wip/laravel-6/src/Auth/Models/User.php#L27-L29
They are some pretty conservative rules.

Reply:
@bennothommo hmm, I would say so yes. That might break some uses but nothing should really use the base model unmodified anyways